version_info_mcpserver.json

```diff
@@ -1,6 +1,6 @@
 {
   "ansible_mcpserver_service": {
-    "imageTag": "0.9.202510071304"
+    "imageTag": "latest"
```

Comment:
This is a temporary change for loading `quay.io/goneri/aap-mcp-downstream:latest`